packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -732,7 +846,8 @@ class _IdentityThemeDataCacheKey {
     // We are explicitly ignoring the possibility that the types might not
     // match in the interests of speed.
     final _IdentityThemeDataCacheKey otherKey = other;
-    return identical(baseTheme, otherKey.baseTheme) && identical(localTextGeometry, otherKey.localTextGeometry);
+    return identical(baseTheme, otherKey.baseTheme) &&
+        identical(localTextGeometry, otherKey.localTextGeometry);
```

Comment:
does this lack of vertical alignment comes from dartfmt, perchance?

Reply:
Yes, it is indenting a continued line by 4 spaces.  Changing the line length to 200 seems to have solved that problem.

Reply:
fwiw, i don't mind it being wrapped, i just want it to be more readable by keeping the vertical alignment consistent if it's wrapped